charts/base-cluster/values.yaml

```diff
@@ -43,6 +43,8 @@ global:
       condition: "{{ .Values.backup.enabled }}"
       additionalLabels:
         app.kubernetes.io/component: backup
+    weave:
+      condition: "{{ .Values.weaveworks.enabled }}"
```

Comment:
Mh, really into it's own namespace?

I would rather put this in flux-system 🤔

Reply:
done